Cython/CodeWriter.py

```diff
@@ -546,7 +546,7 @@ class ExpressionWriter(TreeVisitor):
     def __init__(self, result=None):
         super(ExpressionWriter, self).__init__()
         if result is None:
-            result = u""
+            result = LinesResult()
```

Comment:
Something didn't expect this API change. It introduced lots of test failures.

Reply:
We will pick up this ticket soon.